### PR TITLE
adding the icons

### PR DESCRIPTION
**Pull request by marinanicolai**
@nicolad I couldn't install react-icons or either iconify. Can you please help me figure out why?

### CODE REVIEW

**Comment by nicolad** [comment]
Please provide error

**Comment by marinanicolai** [comment]
@nicolad 
30875 verbose stack TypeError: **Cannot read properties of null (reading 'matches')**
30875 verbose stack     at Link.matches (/home/marina/.nvm/versions/node/v16.14.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/node.js:1090:41)
30875 verbose stack     at Link.canDedupe (/home/marina/.nvm/versions/node/v16.14.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/node.js:1044:15)
30875 verbose stack     at PlaceDep.pruneDedupable (/home/marina/.nvm/versions/node/v16.14.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:465:14)
30875 verbose stack     at PlaceDep.placeInTree (/home/marina/.nvm/versions/node/v16.14.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:326:14)
30875 verbose stack     at PlaceDep.place (/home/marina/.nvm/versions/node/v16.14.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:214:10)
30875 verbose stack     at new PlaceDep (/home/marina/.nvm/versions/node/v16.14.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:71:10)
30875 verbose stack     at /home/marina/.nvm/versions/node/v16.14.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:949:31
30875 verbose stack     at Array.map (<anonymous>)
30875 verbose stack     at Arborist.[buildDepStep] (/home/marina/.nvm/versions/node/v16.14.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:949:8)
30875 verbose stack     at async Arborist.buildIdealTree (/home/marina/.nvm/versions/node/v16.14.0/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:216:7)
30876 verbose cwd /home/marina/next-with-turbo-emotion-theme
30877 verbose Linux 5.15.0-58-generic
30878 verbose argv "/home/marina/.nvm/versions/node/v16.14.0/bin/node" "/home/marina/.nvm/versions/node/v16.14.0/bin/npm" "i" "react-icons"
30879 verbose node v16.14.0
30880 verbose npm  v8.3.1
30881 error Cannot read properties of null (reading 'matches')
30882 verbose exit 1
30883 timing npm Completed in 487621ms
30884 verbose unfinished npm timer reify 1674331469132
30885 verbose unfinished npm timer reify:loadTrees 1674331469147
30886 verbose unfinished npm timer idealTree:buildDeps 1674331481645
30887 verbose unfinished npm timer idealTree:node_modules/.pnpm/babel-plugin-macros@3.1.0/node_modules/babel-plugin-macros 1674331486623
30888 verbose unfinished npm timer idealTree:node_modules/.pnpm/@swc+core@1.3.24/node_modules/@swc/core 1674331763592
30889 verbose unfinished npm timer idealTree:node_modules/.pnpm/ajv@6.12.6/node_modules/ajv 1674331781180
30890 verbose unfinished npm timer idealTree:node_modules/.pnpm/ansi-styles@4.3.0/node_modules/cli-truncate 1674331785603
30891 verbose unfinished npm timer idealTree:node_modules/.pnpm/cosmiconfig@7.1.0/node_modules/cosmiconfig 1674331829638
30892 verbose unfinished npm timer idealTree:node_modules/.pnpm/debug@2.6.9/node_modules/debug 1674331843618
30893 verbose unfinished npm timer idealTree:node_modules/.pnpm/debug@3.2.7/node_modules/debug 1674331844734
30894 verbose unfinished npm timer idealTree:node_modules/.pnpm/debug@4.3.4/node_modules/xo 1674331845771
30895 verbose unfinished npm timer idealTree:node_modules/.pnpm/dir-glob@3.0.1/node_modules/ava 1674331850832
30896 verbose unfinished npm timer idealTree:node_modules/.pnpm/dom-serializer@1.4.1/node_modules/cheerio 1674331854605
30897 verbose unfinished npm timer idealTree:node_modules/.pnpm/domutils@2.8.0/node_modules/domutils 1674331856431
30898 verbose unfinished npm timer idealTree:node_modules/.pnpm/error-ex@1.3.2/node_modules/load-json-file 1674331861779
30899 verbose unfinished npm timer idealTree:node_modules/.pnpm/esquery@1.4.0/node_modules/esquery 1674331909110
30900 verbose unfinished npm timer idealTree:node_modules/.pnpm/esrecurse@4.3.0/node_modules/eslint 1674331911125
30901 verbose unfinished npm timer idealTree:node_modules/.pnpm/file-entry-cache@6.0.1/node_modules/file-entry-cache 1674331925681
30902 verbose unfinished npm timer idealTree:node_modules/.pnpm/for-each@0.3.3/node_modules/for-each 1674331931778
30903 verbose unfinished npm timer idealTree:node_modules/.pnpm/glob@7.1.6/node_modules/glob 1674331939581
30904 verbose unfinished npm timer idealTree:node_modules/.pnpm/glob@7.1.7/node_modules/glob 1674331940103
30905 verbose unfinished npm timer idealTree:node_modules/.pnpm/glob@7.2.3/node_modules/glob 1674331940144
30906 verbose unfinished npm timer idealTree:node_modules/.pnpm/globals@13.19.0/node_modules/xo 1674331941816
30907 verbose unfinished npm timer idealTree:node_modules/.pnpm/globby@11.1.0/node_modules/globby 1674331942056
30908 verbose unfinished npm timer idealTree:node_modules/.pnpm/has@1.0.3/node_modules/has 1674331952031
30909 verbose unfinished npm timer idealTree:node_modules/.pnpm/human-signals@2.1.0/node_modules/human-signals/node_modules/ava 1674331956405
30910 verbose code 1
30911 error A complete log of this run can be found in:
30911 error     /home/marina/.npm/_logs/2023-01-21T20_04_28_875Z-debug-0.log

**Comment by nicolad** [comment]
The import was wrong, check docs:


https://react-icons.github.io/react-icons/icons?name=io5

**Comment by nicolad** [comment]
image

File: apps/expense/components/InfoArea/InfoArea.tsx [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+import {
+  IoArrowForwardCircleSharp,
+  IoArrowBackCircleSharp,
+} from "react-icons/io5";
 import { currentMonthFormat } from "../../helpers/dateFilter";
 import { Layout, Box, Container } from "@shared/core";
 
-
 type Props = {
-  currentMonth: string,
```
Comment (nicolad) [inline apps/expense/components/InfoArea/InfoArea.tsx:7]: @marinanicolai please install prettier, formatting was off

**Pull request merged**